File: Java/Linked List/Creation of Linked List.java
class LinkedList {
	Node head; 
	static class Node {
		int data;
		Node next;
		Node(int d)
		{
			data = d;
			next = null;
		} 
	}

    public void printList()
    {
        Node temp = head;
        while (temp != null) {
            System.out.print(temp.data + " ");
            temp = temp.next;
        }
    }

	public static void main(String[] args)
	{
		LinkedList llist = new LinkedList();
		llist.head = new Node(1);
		Node second = new Node(2);
		Node third = new Node(3);
		llist.head.next = second; 
		second.next = third; 
		llist.printList();
	}
}

File: C++/Linked List/Creation of Linked List.cpp
#include <bits/stdc++.h>
using namespace std;

class Node {
public:
	int data;
	Node* next;
};

void printList(Node* head)
{
	Node* temp = head;
	while (temp != NULL) {
		cout << temp->data << " ";
		temp = temp->next;
	}
}

int main()
{
	Node* head = new Node();
	Node* second = new Node();
	Node* third = new Node();

	head->data = 1; 
	head->next = second; 

	second->data = 2;
	second->next = third;

	third->data = 3; 
	third->next = NULL;
	printList(head);

	return 0;
}


